azure-pipelines.yml

```diff
@@ -42,6 +59,11 @@ stages:
         packageType: sdk
         version: 7.0.x
 
+    - name: Install DotNet workloads
+      shell: bash
+      run: |
+        dotnet workload install android ios tvos macos maui maccatalyst
```

Comment:
I guess this fixes the build issue for WinUI?

Reply:
Strangely the build didn't trigger, I wasn't sure if your using a private build server that may have been off at the time of checkin. But I was testing the ability to build MAUI projects on the server. You may have more information about what happened visible to you as I don't have any permissions for this repository.

Reply:
I was thinking that the MAUI workload would add WinUI and that the others allow us to build a more variant set of cross platform extensions.

Reply:
@Lakritzator Any ideas what is happening with the build server?
Are you using Azure DevOps build servers for any particular reason?

Reply:
The build issues are resolved now, we can merge this 👍